source/about/mattermost-v10-changelog.md

```diff
@@ -3,6 +3,109 @@
+ - Fixed the height of the automatic replies text area to align with proper dimensions.
+ - Fixed various accessibility issues in the User Groups modals.
+ - Fixed accessibility issues in Profile Settings to enhance usability.
+ - Fixed dialog dropdowns to ensure they were not cut off visually.
+ - Fixed an issue where deactivated users still maintained their previous status.
+ - Fixed an issue with plugin dialogs triggering errors upon submission.
+ - Fixed the version number of Support Packet v2.
+ - Fixed an issue with MIME type detection for video files (e.g., MP4, MOV, AVI, WEBM, WMV, MKV, MPG/MPEG) uploaded to S3 storage, enabling inline video playback in browsers.
+ - Fixed Support Packet caching issues.
+ - Fixed the **Threads** textbox changing width when a message was typed on certain screens.
+ - Fixed an issue with log messages including blank "user_id" field when request session was not valid or had a blank user_id.
+ - Fixed an issue where the emoji picker focus did not return to button when not selecting an emoji.
+ - Fixed the label in notification settings for the notification sound combo box.
+
+### config.json
+New setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``ExperimentalSettings`` in ``config.json``:
+    - Added ``ExperimentalChannelCategorySorting`` configuration setting to add the ability to automatically sort channels into categories upon creation/renaming.
+
+#### Changes to Enterprise plan:
+ - Under ``DataRetentionSettings`` in ``config.json``:
+    - Added ``PreservePinnedPosts`` configuration setting. If it's set to ``true``, pinned posts will not be deleted by data retention. 
+
+#### Changes to Enterprise Advanced plan:
+ - Under ``ConnectedWorkspacesSettings`` in ``config.json``:
+    - Added ``MemberSyncBatchSize``, ``SyncUsersOnConnectionOpen``, ``GlobalUserSyncBatchSize`` configuration settings to allow remote users to be discoverable in the Direct/Group Message modal. 
+
+### API Changes
+ - Added property fields and value methods to the plugin API.
+ - Improved the semantics of Groups API errors when invalid parameters were specified.
+
+### Open Source Components
+ - Added ``mholt/archives`` and removed ``code.sajari.com/docconv`` from https://github.com/mattermost/mattermost.
+
+### Go Version
+ - v10.10 is built with Go ``v1.24.3``.
+
+### Known Issues
+ - Setting the license file location through an environment variable still gives the option to upload a new license through the System Console, resulting in the license being overwritten by the one set through the environment variable. See this [knowledge base article](https://support.mattermost.com/hc/en-us/articles/33911983851284-System-console-still-displays-old-license-after-uploading-a-new-one) on how to resolve this issue.
+
```

Comment:
I think I asked about this last time, but "why this issue?" -- surely there are hundreds of known issues that we've left unresolved, and I honestly don't think anyone is even working on this one. Should we just remove `Known Issues`?
```suggestion
```